libdfu/dfu-device.c

```diff
@@ -665,7 +684,7 @@ dfu_device_set_quirks (DfuDevice *device)
 		priv->quirks |= DFU_DEVICE_QUIRK_IGNORE_INVALID_VERSION;
 
 	/* m-stack DFU implementation */
-	if (vid == 0x273f && pid == 0x1003)
+	if (vid == 0x273f && (pid == 0x1003 || pid == 0x100a))
```

Comment:
New PID should probably come in a separate commit from this.

Reply:
Ohh it did, sorry, i pushed the PID change to master already. This hunk should just disappear on rebase.